example/__tests__/setup/elements.js

```diff
@@ -0,0 +1,26 @@
+import helper from 'tipsi-appium-helper'
```

Comment:
I think this file is overhead for our tests, remove it and move all ids in test file